Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -250,9 +248,11 @@ - (void)updateDateHeader
             [self.dateHeaderView autoPinLeadingToSuperviewMarginWithInset:self.conversationStyle.gutterLeading],
             [self.dateHeaderView autoPinTrailingToSuperviewMarginWithInset:self.conversationStyle.gutterTrailing],
             [self.dateHeaderView autoPinEdgeToSuperviewEdge:ALEdgeTop],
-            [self.dateHeaderView autoSetDimension:ALDimensionHeight toSize:self.dateHeaderHeight],
 
-            [self.messageBubbleView autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:self.dateHeaderView],
+            [self.messageBubbleView autoPinEdge:ALEdgeTop
```

Comment:
It seems like there's some subtlety behind this change. This looks equivalent, but less to the point. Why this change?

Reply:
Added a comment:

```
            // DO NOT pin to the bottom of dateHeaderView.
            //
            // Being a UIStackView, it doesn't reflect the spacing below the date
            // header contents.  Instead pin using dateHeaderHeight which includes
            // the spacing.
```